### PR TITLE
Show how to use the Q calculation engine of E4CV

### LINKED ISSUE #138: How to access the other libhkl engines such as incidence or Q?
Such as the Engines under the E4CV documentation:
https://people.debian.org/~picca/hkl/hkl.html#org7ef08ba

### PR DESCRIPTION
**Pull request by prjemian**
Demonstrate how to use a different calculation engine other than the default *hkl*.
FIX #138

### CODE REVIEW

**Comment by prjemian** [comment]
@jwkim-anl, @gfabbris, @strempfer, @rodolakis: You might be interested in this.  See: https://github.com/bluesky/hklpy/blob/138-other-engines/examples/var_engines.ipynb

**Review by mrakitin — COMMENTED** [review]
Looks good, have a single comment about units that we discussed in another PR.

File: examples/var_engines.ipynb [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+    "\n",
+    "NOTE:  The choice of calculation engine is locked in the `hkl.diffract.Diffractometer()` class.  Once the diffractometer object is created, the calculation engine cannot be changed.\n",
+    "\n",
+    "**Objective**\n",
+    "\n",
+    "Many of the examples in the *hklpy* repository use the *hkl* engine, it is the most common use case.  Below, we'll demonstrate the `q` calculation engine of the `E4CV` (4-circle Eulerian in vertical scattering) geometry.\n",
+    "\n",
+    "## Standard Imports\n",
+    "\n",
+    "First, we start by importing the `gi` ([*gobject-introspection*](https://pygobject.readthedocs.io/en/latest/index.html)) package which is required to link the *libhkl* library (`'Hkl', 5.0`) into Python.  Then import the voltage wavelength product, `A_KEV`.  (Keep in mind that `gi` is not the `Gtk` library, just the tools that enable other languages such as Python to use the libraries compiled with [Gnome's Glib Object: GObject](https://developer.gnome.org/gobject/stable/) framework.)"
```
[the remark this answers is not in the document]
Comment (prjemian) [reply examples/var_engines.ipynb]: Copied text that was used in PR #153.

**Comment by prjemian** [comment]
@mrakitin: Can we finalize this for merge by next week (6/14)?

**Review by mrakitin — APPROVED** [review]
@prjemian, sorry for the delay. Looks good to me!

**Comment by prjemian** [comment]
@mrakitin Thanks for the review!

**Pull request merged**